polling/periodic_sequence/child_workflow.go

```diff
@@ -0,0 +1,39 @@
+package periodic_sequence
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+type ChildWorkflowParams struct {
+	SingleWorkflowAttempts int
```

Comment:
I think SingleWorkflowAttempts is an implementation detail of the child. So there is no reason for the parent to set it.

Reply:
👍 I thought it could be useful for unit testing purposes to have control around it, but in reality controlling the polling interval is more important to have unit tests that execute quickly.  This was before I added the PollingDuration to the struct. 

Do you think hardcoding in the child workflow is appropriate? Making it private and default it via constructor-like factory function would not work, I believe, because it won't be serialized